### DISCUSSION TITLE
[Question]: Issue getting social logins working.. Please see pictures in post.

### DISCUSSION THREAD

**Discussion by bpawnzZ** [danny-avila/LibreChat#3453] [Troubleshooting]
### What is your question?

I am trying to get Social logins to work.  I have fully setup Oath for google and github.  However when i click on the login with social provder button i get redirected to domain not found/browser error.  But if i mouse over the button I can clearly see that the it inserts the domain twice which throws off the button from working obviously.  I have set my domain in 'DOMAIN_CLIENT' and 'DOMAIN_SERVER' does anyone have any idea what i am doing wrong or how i should proceed.  For my reverse proxy I am using Nginx.  Not sure if that is really affecting anything or not.  Thank you for your help!  Please see pics for clarification on what i am describing.

### More Details

I have fully followed all instruction in the Librechat docs to set this up.  Don't understand what I am missing.

### What is the main subject of your question?

User System/OAuth

### Screenshots

![screenshotzoom](https://github.com/user-attachments/assets/fa2f67ae-abc1-4bf6-a56b-e3c7436a113d)
![screenshotupdate2](https://github.com/user-attachments/assets/c85354a2-0e00-465f-a089-3538978a3a44)
![screenshot_25072024_013202](https://github.com/user-attachments/assets/5058b448-5ac1-4438-847a-2d2075732ec3)
![screenshot_25072024_012347](https://github.com/user-attachments/assets/e70bcaac-41fe-4d3b-a868-06ac20a1f3c7)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bpawnzZ** [comment]
If anyone could help I'd appreciate it

**Comment by bpawnzZ** [comment]
Tons of support i see

**Reply by berry-13** [reply]
Well, the discussion is closed. We likely missed this discussion. Was it already closed before sending the other replies? Anyway, yes, it seems like a reverse proxy issue. Would you mind checking the Inspect Element tab and seeing the redirects your browser is doing? If the Google Auth and callback URL are set up correctly, everything should work, double-check check the redirect url in the google auth setup page
